device-types/HPE/OfficeConnect-1920-8G-PoE-180W.yml

```diff
@@ -0,0 +1,57 @@
+---
+manufacturer: HPE
+model: OfficeConnect 1920-8G-PoE-180W
+slug: hpe-officeconnect-1920-8g-poe-180w
+part_number: JG922A
+u_height: 1
+is_full_depth: false
+front_image: false
+rear_image: false
+comments: '[OfficeConnect 1920 Switch Series - Specifications](https://support.hpe.com/hpesc/public/docDisplay?docId=c04478097&docLocale=en_US)'
+weight: 3.2
+weight_unit: kg
+airflow: passive
+power-ports:
+  - name: PS1
+    type: iec-60320-c14
+    maximum_draw: 235
+console-ports:
+  - name: Console
+    type: rj-45
+interfaces:
+  - name: GE1/0/1
+    type: 1000base-t
+    poe_mode: pse
+    poe_type: type1-ieee802.3af
```

Comment:
Looking at the spec sheet this switch is POE+ therefore the correct poe_type is `type2-ieee802.3at`